lib/stack.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
At a certain point, you stopped tracking the /lib folder in your git ignore, but these files are still hanging around. In the future, I challenge you to up your git-fu to alter the commit that added the files.